docs/code-snippets/outlook-snippets.yaml

```
@@ -1921,6 +1921,42 @@ Office.SpamReportingEventCompletedOptions:interface:
             });
         });
     }
+Office.SpamReportingEventCompletedOptions#commandId:member:
+  - |-
+    function onSpamReport(event) {
+        // Run operations to process the reported message.
+
+        /**
+        * Signals that the spam-reporting event has completed processing.
+        * A task pane is opened instead of showing a post-processing dialog.
+        * Context data is then passed to the task pane for processing.
+        * To ensure that the task pane opens and receives the context data,
+        * the reported message mustn't be moved from the folder in which it resides.
+        */
```

should indent - here and other instance

```suggestion
        /**
         * Signals that the spam-reporting event has completed processing.
         * A task pane is opened instead of showing a post-processing dialog.
         * Context data is then passed to the task pane for processing.
         * To ensure that the task pane opens and receives the context data,
         * the reported message mustn't be moved from the folder in which it resides.
         */
```